packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.test.js

```diff
@@ -255,9 +255,9 @@ describe('useAutocomplete', () => {
     };
 
     const devErrorMessages = [
-      "Error: Uncaught [TypeError: Cannot read property 'removeAttribute' of null]",
+      "Error: Uncaught [TypeError: Cannot read properties of null (reading 'removeAttribute')]",
```

Comment:
Please revert these changes. We're running our tests under Node 12. A more recent version of Node changed the error message (that's why likely these tests fail on your machine but pass on CI). We will update the error messages once we configure the CI to run on a newer Node.

Reply:
yeah, I noticed they were making another test fail even in the CI that's why I was confused. It seems like when I change it, the `ci/circleci: test_unit-1`  started to fail (I think it was the `test fake browser` step), but the other test was passing. I'm used to working on gitlab so still getting use to the CI/CD flow on github. Not sure how to see the checks that passed and failed from a previous commit. But, I changed this back.

Reply:
I just found a way to see the previous pipeline that failed. It seems like the same test failed for a different reason and when I ran locally I saw the failure above that I ended up changing. The initial failure was probably due to this change somehow, not 100% sure why yet. 

Working on updating the tests for this. I'm thinking to test this I have to add a test that has a function that returns the scenario above (i.e. `<Autocomplete/>` that returns a  `<TextArea/>` or `<TextField/>` with `multiline` prop set to `true` and uses the `useAutocomplete()` hook) to make sure the warning message is getting output instead of the error message.